### DISCUSSION TITLE
Group inside a Date picker popover is not working properly

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#7332] [Q&A] [answered]
I've developed a date picker component using React Aria components. I have a use case where I need to display date fields within the date picker's dialog popover. To fulfill my requirements, I'm using the **<Group />** component from React Aria Components to include two date field components. However, when I place the **<Group />** component inside the date picker dialog popover, the popover's position behaves unexpectedly and does not align as intended.

You can check the issue here -> https://codesandbox.io/p/sandbox/cz4n2y?file=%2Fsrc%2FDatePicker.tsx%3A71%2C1

Another data point is when i place Group inside normal popover with its working fine you can check that too in above link.

**Comment by snowystinger** [comment, marked as answer]
I think this is the same issue as https://github.com/adobe/react-spectrum/issues/6934#issuecomment-2309125809

Here's it working https://codesandbox.io/p/sandbox/rac-date-picker-forked-2ks3ng

**Reply by RajaTheKing826** [reply]
Yes, it worked for this problem.
